### PR TITLE
Add Neopets

### PR DESCRIPTION
**Pull request by Yang-Nankai**
Neopets is a virtual pet website that allows users to adopt, care for, and interact with digital pets in a fantasy world. The platform features games, quests, and an in-game economy where players can buy, sell, and trade items. Neopets also provides a social experience with community features and events. The platform offers 2FA for the account login verification process. The website is ranked 3,195 on [[Similarweb](https://pro.similarweb.com/#/digitalsuite/websiteanalysis/overview/website-performance/*/999/1m?webSource=Total&key=neopets.com)](https://pro.similarweb.com/#/digitalsuite/websiteanalysis/overview/website-performance/*/999/1m?webSource=Total&key=neopets.com). The website provides documentation on how to turn on 2FA.

ps: Below is the 2FA authentication triggered during the login process.
image

### CODE REVIEW

File: img/n/neopets.com.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="84" height="68" fill="none"><g fill="#FECF48" clip-path="url(#clip0_1270_6759)"><path d="M4.27 9.377c1.693 2.14 3.534 4.356 5.448 6.645l-7.73 16.76 14.797-8.417c2.724 3.175 5.89 6.94 9.497 11.37l-3.828-14.471 9.203-4.8-11.412-4.208L17.08 0l-4.197 9.303L0 4.356l4.27 5.02ZM62.724 21.19a639.345 639.345 0 0 0-1.988 11.74l-24.441 7.014 22.085 7.9c-.81 5.685-1.914 12.404-3.239 20.156l11.706-17.055 13.251 5.168-5.89-15.727 9.866-14.25L70.6 30.125 64.05 12.182l-1.325 9.008Z"/></g><defs><clipPath id="clip0_1270_6759"><path fill="#fff" d="M0 0h84v68H0z"/></clipPath></defs></svg>
```
Comment (Carlgo11) [inline img/n/neopets.com.svg:1]: Replacing height/width with viewBox like the test suggested.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 84 68"><path fill="#fecf48" d="M4.3 9.4 9.7 16 2 32.8l14.8-8.4 9.5 11.3-3.8-14.4 9.2-4.8-11.5-4.2L17.1 0l-4.2 9.3L0 4.3l4.3 5Zm58.4 11.8a639.3 639.3 0 0 0-2 11.7l-24.4 7 22 8A397 397 0 0 1 55.1 68l11.7-17 13.3 5-5.9-15.7L84.1 26l-13.5 4-6.5-18-1.4 9"/></svg>
```

File: entries/n/neopets.com.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "NeoPass": {
+    "domain": "neopets.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://neopass.support.neopets.com/hc/articles/25933646591501",
+    "categories": [
+      "entertainment"
```
Comment (Carlgo11) [inline entries/n/neopets.com.json]: Is it entertainment of an online game? I've heard the name Neopets before but I'm not quite sure what it is.

File: entries/n/neopets.com.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "NeoPass": {
```
Comment (Carlgo11) [inline entries/n/neopets.com.json]:
```suggestion
  "Neopets": {
```

Comment (Yang-Nankai) [reply entries/n/neopets.com.json]: Thanks for your correction, I misread the title of the page, it is indeed **Neopest**. 👍

File: entries/n/neopets.com.json (hunk 2 above)
Comment (Yang-Nankai) [reply entries/n/neopets.com.json]: The following explanation comes from [Wikipedia](https://en.wikipedia.org/wiki/Neopets):

> Neopets is a [free-to-play](https://en.wikipedia.org/wiki/Free-to-play) [virtual pet](https://en.wikipedia.org/wiki/Virtual_pet_site) [browser game](https://en.wikipedia.org/wiki/Browser_game). First launched in 1999, the game allows users to own virtual pets ("Neopets") and explore a virtual world called "Neopia." Players can earn one of two [virtual currencies](https://en.wikipedia.org/wiki/Virtual_currency). One currency, called Neopoints, can be obtained for free through on-site features like games, events, and contests. The other, Neocash (NC), is purchased with real-world money and can be exchanged for wearable items for pets.

Comment (hkamran80) [reply entries/n/neopets.com.json]: I think the gaming category is better.

Comment (Yang-Nankai) [reply entries/n/neopets.com.json]: @hkamran80  Thanks! I missed the `gaming` category. ❤️

**Review by fletcherdodd — APPROVED** [review]

**Pull request merged**